index.html

```diff
@@ -73,61 +73,73 @@ <h1 class="title">SIMPLE HOURLY RATES</h1>
           <p>We aim to deliver the best customer service to our customers. It is our promise.</p>
         </div>
       </section>
-    </div>
+</div><!-- End Section 2 Why VFix -->
 
 
       <section class="flexbox-grid">
-        <div><h1>Simple Pricing $59.99 per hour. No hidden fees. Period.</h1></div>
+        <div><h1 class="tagline">Simple Pricing $59.99 per hour. No hidden fees. Period.</h1></div>
       </section>
 
       <section>
-        <div class="services">
+        <a name="services"><div class="services"></a>
+
           <div class="service-item">
-            <img src="images/virus-removal.png" alt="money">
-            <h2>Virus Removal</h2>
-            <p>Our simple hourly rates make our technical support easy and affordable for everyone.</p>
+            <div><h1 class="services-title">SERVICES</h1></div>
+            <img class="image-icons" src="images/virus-removal.png" alt="money">
+            <h2>VIRUS REMOVAL</h2>
+            <p>We’ll sweep and check your PC for viruses and eliminate any possible threats that could be slowing you down. We will also focus on maintaining your PC’s security by taking care of viruses early to avoid file loss and data corruption.</p>
           </div>
 
         <div class="service-item">
+          <br><br>
```

Comment:
There's lots of these `<br><br>`.  Two things:
* As a rule, we should close all tags, even those that are "self-closing".  This should therefore be `<br /><br />`
* We should not use `<br>` for padding - we have CSS for that 😛